Makefile

```diff
@@ -27,4 +27,20 @@ all-tests:
 generate build clean:
 	go $@ $(EXTRAGOARGS)
 
+sandbox-test-fc-build:
+	docker build -f fctesting/sandbox/Dockerfile -t "firecracker" .
```

Comment:
Let's not call the image "firecracker".  How about "localhost/firecracker-go-sdk-sandbox`?